Source/WebKit/Scripts/webkit/tests/GeneratedSerializers.cpp

```diff
@@ -515,6 +516,76 @@ std::optional<WebCore::FloatBoxExtent> ArgumentCoder<WebCore::FloatBoxExtent>::d
     };
 }
 
+enum class WebCore_TimingFunction_Subclass : uint8_t {
```

Comment:
I would expect an isValidEnum function to be generated for this so the decoder works.